src/main/configurations/Translate/Configuration_ResolveProfileValues.xml

```diff
@@ -0,0 +1,62 @@
+<Module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../FrankConfig.xsd">
+    <Adapter name="ResolveProfileValues"
+        active="${ResolveProfileValues.Active}"
+        description="">
+
+        <Receiver name="ResolveProfileValues">
+            <JavaListener name="ResolveProfileValues" returnedSessionKeys="Error" />
+        </Receiver>
+
+        <Pipeline>
+            <Exits>
+                <Exit name="EXIT" state="SUCCESS" />
+                <Exit name="EXCEPTION" state="ERROR" />
+            </Exits>
+            
+            <Cache />
+            <LocalFileSystemPipe name="ReadFile" action="read">
+                <Param name="filename" value="${configurations.directory}/Translate/Profiles.json" />
+                <Forward name="success" path="IsXml" />
+            </LocalFileSystemPipe>
+
+            <XmlWellFormedCheckerPipe name="IsXml">
+                <Forward name="success" path="EXCEPTION" />
+                <Forward name="failure" path="JsonToXml" />
+            </XmlWellFormedCheckerPipe>
+
+            <JsonPipe name="JsonToXml" storeResultInSessionKey="profiles"/>
+
+            <ForEachChildElementPipe name="AddPathAttributeToEachProfile"
+                getInputFromSessionKey="profiles" targetElement="profile">
+                <XsltSender styleSheetName="Common\xsl\AddPathAttribute.xsl" indentXml="true"
+                    omitXmlDeclaration="true">
+                    <Param name="baseProfile" sessionKey="baseProfile" type="DOMDOC" />
+                </XsltSender>
+            </ForEachChildElementPipe>
+
+            <XsltPipe name="RemoveResultTags" styleSheetName="Common\xsl\RemoveResultTags.xsl" storeResultInSessionKey="profiles"/>
+
+            <XsltPipe name="ExtractBaseProfile" styleSheetName="Common\xsl\ExtractBaseProfile.xsl"
+                omitXmlDeclaration="true" storeResultInSessionKey="baseProfile" outputType="XML">
+            </XsltPipe>
+
+            <!-- Failure here indicates missing baseProfile, or more than one baseProfile -->
+            <XmlWellFormedCheckerPipe name="checkBaseProfile">
+                <Forward name="success" path="AddBaseProfileToEachProfile" />
+                <Forward name="failure" path="EXCEPTION"/>
```

Comment:
configuring a base profile should be optional. If there is no base profile there is nothing to merge, so it can just go to EXIT with the unedited profiles.